src/classes/tool_handlers.py

```
@@ -3766,6 +3766,48 @@ def _refresh_thumb():
+        ai["description"] = summary[:400]
+        # analyzed=True so get_effective_ai_metadata / Scene panel show the text.
+        ai["analyzed"] = True
+        ai["source"] = "ai_video_generation"
+        file_obj.data["ai_metadata"] = ai
+        if not file_obj.data.get("name"):
+            file_obj.data["name"] = summary[:120]
+        file_obj.save()
+        try:
+            _get_app().window.FileUpdated.emit(str(file_obj.id))
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)